### ISSUE TITLE
About quantity

### ISSUE THREAD

**Issue by youngbo9i** [catly/drug-target#2]
A total of 2664 samples were selected. Why is there a total of 2222 samples in the training set and test set?
